src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FluidPump.java

```diff
@@ -52,18 +56,24 @@
     private static final int ENERGY_CONSUMPTION = 32;
     private static final int RANGE = 42;
 
+    private final ItemStack WATER_BOTTLE = new ItemStack(Material.POTION);
```

Comment:
I probably confused things here by adding an alternatiuve - If a field is static then it needs to be in SCREAMING_SNAKE_CASE but if not, then camelCase (waterBottle) is fine. Likewise if a field is a static constant then it goes above all other declared fields, if it is final then it goes with other private finals and, in this case, below emptyBottle - My question here was whether it needed to be static or not. 

I would now move this below emptyBottle, change to waterBottle and keep it non-static as you have it currently

Reply:
im the one who got confused sefi, my bad xD. will do the changes in a moment. I really need to review conventions and standards